html/elements/textarea.json

```diff
@@ -545,8 +545,7 @@
                   "version_added": "12"
                 },
                 "firefox": {
-                  "version_added": false,
-                  "notes": "See <a href='https://bugzil.la/1312897'>bug 1312897</a>."
+                  "version_added": "59"
                 },
                 "firefox_android": {
                   "version_added": false,
```

Comment:
This looks to have been fixed in Android too. Could you set the same version number and remove the note here too?